README.md

```diff
@@ -282,7 +282,7 @@ And finally, you are ready to go! Or at least you should be...
 **Iris Mestres** | [Contact me][deliaBlue]
 
 ### Junior Contributors
-
+**Paula Martín** | [Contact me][paulamartin14] | I haven't slept much tonight
 
 
 
```

Comment:
Additionally the link source to you git profile is missing, at the end of the document add:

[paulamartin14]:< ... >

where instead of ... you out the link to your git profile.